Mixpanel/Persistence.swift

```diff
@@ -118,8 +118,21 @@ class Persistence {
 
         if !NSKeyedArchiver.archiveRootObject(object, toFile: path) {
             Logger.error(message: "failed to archive \(type.rawValue)")
+            return
         }
 
+        addSkipBackupAttributeToItem(at: path)
+    }
+
+    class private func addSkipBackupAttributeToItem(at path: String) {
+        var url = URL.init(fileURLWithPath: path)
+        var resourceValues = URLResourceValues()
+        resourceValues.isExcludedFromBackup = true
```

Comment:
what's the reason behind skipping backups? feels like backups might be nice to transfer things like super props to new phones. probably don't want backups for queue archives though